### ISSUE TITLE
Core Dumped

### ISSUE THREAD

**Issue by sobotka** [slowmoVideo/slowmoVideo#37]
````
Rendering frame number  248  @ 10.3333  from source time  1.81898
Re-using existing flow image for left frame 30 to right frame 31: "Documents/slow2/cache/oFlowOrig/ocv-forward-30-31.sVflow"
Read flow file of size 1920×816. Magic number: flow_sV, version: 1
Re-using existing flow image for left frame 31 to right frame 30: "Documents/slow2/cache/oFlowOrig/ocv-backward-31-30.sVflow"
Read flow file of size 1920×816. Magic number: flow_sV, version: 1
  Saved frame number  248  to  "/tmp/slowmoUI-oNdDIv/rendered-00249.png"
Rendering frame number  249  @ 10.375  from source time  1.82631
Building flow for left frame  31  to right frame  32 ; Size:  Original frame size
calcOpticalFlowDual_TVL1
OpenCV Error: Assertion failed (I0.size() == I1.size()) in calc, file /build/buildd/opencv-2.4.9+dfsg/modules/video/src/tvl1flow.cpp, line 158
terminate called after throwing an instance of 'cv::Exception'
  what():  /build/buildd/opencv-2.4.9+dfsg/modules/video/src/tvl1flow.cpp:158: error: (-215) I0.size() == I1.size() in function calc

Aborted (core dumped)
````

**Comment by valgit** [comment]
Can you provide more information to reproduce the bug ?
for eg: input resolution, number of frame, platform, how you build it .... whatever can be useful !

**Comment by sobotka** [comment]
Let me know when done so I can remove it.

https://s3.amazonaws.com/bug-attachments/slow2.tar.gz

**Comment by valgit** [comment]
Hi,
thanks, I've got it, I will try to find some time to look at the problem ASAP.

**Comment by valgit** [comment]
Hi, I've tried to test your project, but I missed your movie !  So, I try to bypass modifiing your project as a timelapse. But in this case the rendering went well ;-)
so : 
* did you use the last OpenCV version ?
* can you share the input movie ?

**Comment by valgit** [comment]
should be corrected since PR #67
